components/Code/index.tsx

```diff
@@ -86,7 +88,6 @@ const Code = (props: CodeProps) => {
         ref={codeRef}
         className={['rdmd-code', `lang-${language}`, `theme-${theme}`].join(' ')}
         data-lang={language}
-        suppressHydrationWarning={true}
```

Comment:
`suppressHydrationWarning` only works one level deep, so it isn't doing anything for us here. Did it work for an entire subtree before React 18 or something?